dockertest/environment.py

```diff
@@ -79,3 +81,101 @@ def selinux_is_enforcing():
         raise ValueError("Unexpected value from"
                          " security_getenforce(): %s" % mode)
     return mode == selinux.ENFORCING
+
+
+class RPMS(Mapping):
+    """
+    Read-only mapping of rpm names to ``value_type`` instances
+
+    :param args: Same as standard ``dict()`` positional arguments.
+    :param dargs: Same as standard ``dict()`` keyword arguments.
+    """
+
+    #: Resentation for values, the first index must be the package name.
+    value_type = namedtuple('value_type',
```

Comment:
I was originally thinking of having this be a base-class, then specializing it as ``RPMS``.  That's why there's this (and other) generic names used.  Yes, happy to rename, I'm not married to much at all here.